app/views/profiles/_profile.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="medium-12 columns">
+  <h3>Profile</h3>
+    <form data-abide>
+      <div class="row">
+        <div class="medium-12 columns username-field">
+          <label>First Name
+            <input type="text" required />
+          </label>
+          <small class="error">Required</small>
+        </div>
+      </div>
+      <div class="row">
+        <div class="medium-12 columns username-field">
```

Comment:
I'm not sure I understand what these `username-field` classes are for, or why they're repeated. Do we need them?